ci/obmd/run_integration_tests.sh

```diff
@@ -6,4 +6,4 @@ wget https://github.com/CCI-MOC/obmd/releases/download/v0.1/obmd
 chmod +x obmd
 sudo mv obmd /usr/local/bin
 
-py.test tests/integration/obmd.py
+py.test --cov=hil --cov-append tests/integration/obmd.py
```

Comment:
Maybe I missed something, why are we enabling coverage here?

Reply:
we can still see the coverage in the travis logs if we need to, thought it's not really needed.